lib/sensu-plugins-http/version.rb

```diff
@@ -1,8 +1,8 @@
 module SensuPluginsHttp
   module Version
-    MAJOR = 1
+    MAJOR = 2
```

Comment:
Please do not change the version yourself. The sensu-plugins team does that when releasing a new version of the plugin.